tools/linux/uio_drivers/blk/blk.c

```diff
@@ -65,12 +60,13 @@ int driver_init(void **maps, uintptr_t *maps_phys, int num_maps, int argc, char
     blk_req_queue_t *req_queue = (blk_req_queue_t *)maps[1];
     blk_resp_queue_t *resp_queue = (blk_resp_queue_t *)maps[2];
     blk_data = (uintptr_t)maps[3];
-    blk_data_phys = maps_phys[3];
 
+    // @ericc: After changing to using offsets in the sddf protocol, we no longer need to know the 
+    // physical address of any mappings. But now that it is a part of the libuio library, we'll simply do nothing.
```

Comment:
This seems just like a comment explaining things rather than a TODO. If that's the case we should remove the `@ericc`.